src/hooks/useWritingProblems.ts

```diff
@@ -0,0 +1,63 @@
+import { useCallback, useState } from "react";
+import { NativeStackScreenProps } from "@react-navigation/native-stack";
+import { useDispatch } from "react-redux";
+import { HttpMethod, RootStackParamList } from "../types";
+import getProblem from "../assets/prompts";
+
+import gameDescriptions from "../screens/Stacks/gameDescriptions";
+import { completedWriting } from "../redux/reducers/gameDetailsReducer";
+import { internalRequest } from "../requests";
+
+type Props = NativeStackScreenProps<RootStackParamList, "WritingMain">;
+
+const TOTAL_TIME = gameDescriptions.Writing.minutes * 60;
+export default function useWritingProblems({ navigation, route }: Props) {
+  const dispatch = useDispatch();
+  const [problem, setProblem] = useState(getProblem());
+  const [questionsAttempted, setQuestionsAttempted] = useState(0);
+  const [questionsCorrect, setQuestionsCorrect] = useState(0);
+
+  const updateStatsOnAnswer = useCallback((isCorrect: boolean) => {
+    setQuestionsAttempted((prev) => prev + 1);
+    if (isCorrect) {
+      setQuestionsCorrect((prev) => prev + 1);
+    }
+  }, []);
```

Comment:
Based on my previous comment, we do not have a right or wrong answer for the Writing game. We just want to keep track of the number of writing questions they answered. Instead of `questionsAttempted` and `questionsCorrect`, keep track of only `questionsAnswered`, which is incremented every time we get a new problem.

Reply:
Should I delete the other two `questionsAttempted` and `questionsCorrect` variables then?